src/com/ableton/VirtualEnv.groovy

```diff
@@ -0,0 +1,30 @@
+package com.ableton
+
+
+class VirtualEnv implements Serializable {
+  def script
+  def destDir
+
+  VirtualEnv(script, python) {
+    this.script = script
+    def pathSeparator = script.isUnix() ? "/" : "\\"
+    this.destDir = script.pwd(tmp:true) +
+      pathSeparator +
+      script.env.BUILD_NUMBER +
+      pathSeparator +
+      python
+  }
+
+
+  def installRequirements() {
```

Comment:
I think we don't need that function. 
```
venv.installRequirements()
```
vs
```
venv.run("pip install -r requirements.txt")
```
The second version is not that longer than the first one, but way more explicit.

Reply:
Ok, I removed this method.